module-domain/src/main/kotlin/site/yourevents/guest/service/OwnerNicknameService.kt

```diff
@@ -0,0 +1,37 @@
+package site.yourevents.guest.service
+
+import jakarta.transaction.Transactional
+import org.springframework.stereotype.Service
+import site.yourevents.guest.domain.Guest
+import site.yourevents.guest.port.`in`.OwnerNicknameUseCase
+import site.yourevents.guest.port.out.OwnerNicknamePersistencePort
+import site.yourevents.invitation.exception.InvitationNotFoundException
+import site.yourevents.invitation.port.`in`.InvitationUseCase
+import site.yourevents.member.exception.MemberNotFountException
+import site.yourevents.member.port.`in`.MemberUseCase
+import java.util.*
+
+@Service
+@Transactional
+class OwnerNicknameService(
```

Comment:
마찬가지로, 네이밍 수정 건의드리는 바입니다!
```suggestion
class GuestService(
```

Reply:
넵!! 수정하였습니다.